tests/unit/test_0Z_model_validation.py

```diff
@@ -175,23 +175,13 @@ def test_batch_validation(
     for update_data, reference_result in zip(update_list, reference_output_list):
         model_copy = copy(model)
         model_copy.update(update_data=update_data)
-        try:
-            result = calculation_function(
-                model_copy, **supported_kwargs(kwargs=base_kwargs, supported=calculation_args)
-            )
-        except Exception as e:
-            print(f"An error occurred during calculation: {e}")
-        else:
-            compare_result(result, reference_result, rtol, atol)
+        result = calculation_function(model_copy, **supported_kwargs(kwargs=base_kwargs, supported=calculation_args))
+        compare_result(result, reference_result, rtol, atol)
 
     # execute in batch one go
     for threading in [-1, 0, 1, 2]:
         kwargs = dict(base_kwargs, update_data=update_batch, threading=threading)
-        try:
-            result_batch = calculation_function(model, **supported_kwargs(kwargs=kwargs, supported=calculation_args))
-        except Exception as e:
```

Comment:
Was this exception thrown here before because of the incompatibility added initially (not anymore) when the binary search was introduced for the tap changer strategy? So this no longer is needed, right?

Reply:
At Python API level the combination error will never be hit and hence not exposed. This change from the binary PR (https://github.com/PowerGridModel/power-grid-model/pull/684) could potentially hide issues with future added features. In this PR this exception handling logic is removed as a patch to https://github.com/PowerGridModel/power-grid-model/pull/684